ui/src/react-admin/modules/alerts/alerts.const.ts

```diff
@@ -11,3 +18,74 @@ export const ALERTS_QUERY_PARAM_CONFIG = {
 	orderDirection: withDefault(SortDirectionParam, 'asc'),
 	page: withDefault(NumberParam, 1),
 };
+
+// Yup
+
+export const ALERTS_FORM_SCHEMA = (tText: any): SchemaOf<AlertFormState> => {
+	return object({
+		title: string().required(
+			tText('react-admin/modules/alerts/views/alerts-const___titel-is-verplicht')
+		),
+		message: string().required(
+			tText('react-admin/modules/alerts/views/alerts-const___beschrijving-is-verplicht')
+		),
+		fromDate: string().required(
+			tText('react-admin/modules/alerts/views/alerts-const___datum-is-verplicht')
+		),
+		untilDate: string().required(
+			tText('react-admin/modules/alerts/views/alerts-const___datum-is-verplicht')
+		),
+		userGroups: array()
+			.of(string().required('is required'))
+			.min(
+				1,
+				tText(
+					'react-admin/modules/alerts/views/alerts-const___selecteer-een-gebruikersgroep'
+				)
+			),
+		type: string().required(
+			tText('react-admin/modules/alerts/views/alerts-const___icoon-is-verplicht')
+		),
+	});
+};
+
+// Constants
+
+export const alertUserGroups: MultiSelectOption[] = [
+	{
+		label: 'meemoo admin',
+		id: Group.MEEMOO_ADMIN,
+		checked: false,
+	},
+	{
+		label: 'CP admin',
+		id: Group.CP_ADMIN,
+		checked: false,
+	},
+	{
+		label: 'Eindgebruiker',
+		id: Group.VISITOR,
+		checked: false,
+	},
+	{
+		label: 'kiosk',
+		id: Group.KIOSK_VISITOR,
+		checked: false,
+	},
+];
```

Comment:
shouldn't these come from the admin-core-config so we can choose which user groups are shown for avo vs hetarchief?
maybe you can use the same way to list the user groups as for the permissions table?
UserGroupService.fetchUserGroups